pipelines/tekton/aiedge-e2e/aiedge-e2.model-registry.tensorflow-housing.pipelinerun.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  labels:
+    tekton.dev/pipeline: aiedge-e2e
+  generateName: aiedge-e2e-
+spec:
+  params:
```

Comment:
The list of provided param values is not complete, therefore, the pipeline run is not runnable. Parameters without default values must be provided and I have identified that these are missing:
* modelRelativePath
* fetch-model